### PR TITLE
[Needs Review] b07902043 許喬茵

### PR DESCRIPTION
**Pull request by goodyhsu**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## Is there any suggestion?

<br />

### CODE REVIEW

**Comment by goodyhsu** [comment]
@amjltc295 I committed sample_code again. Is this okay now? Thank you.

**Comment by goodyhsu** [comment]
@amjltc295  Is it okay now? Thank you.

File: src/students/b07902043.py [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+    # under src/.
+
+    # You are allowed to change the img_url to your own image URL.
+
+    # Display the image:
+    # result_img.show()
+    # Note: please comment this line when hand in.
+    from PIL import Image, ImageFont, ImageDraw
+
+    def draw_text(
```
Comment (amjltc295) [inline src/students/b07902043.py]: Do not define draw_text() again. Please import it and use it.

Comment (goodyhsu) [reply src/students/b07902043.py]: @amjltc295 Is it okay? Thank you again.

**Comment by amjltc295** [comment]
It would be better to revise your commit message.

**Comment by goodyhsu** [comment]
@amjltc295 Is there still something I need to improve?

**Pull request merged**